From 5f495512685b5496fad1bc994698f3dc41ebdf13 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Sun, 6 Jun 2021 04:17:49 +0530
Subject: [PATCH 059/212] hack! arm64: dts: qcom: sdm845: Add q6voice APR
 service device nodes

arm64: dts: qcom: sdm845: add PDs to q6voice APR

Original author: Joel Selvaraj <jo@jsfamily.in>

Signed-off-by: Casey Connolly <casey.connolly@linaro.org>
Signed-off-by: David Heidelberg <david@ixit.cz>
---
 arch/arm64/boot/dts/qcom/sdm845.dtsi | 34 ++++++++++++++++++++++++++++
 1 file changed, 34 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845.dtsi b/arch/arm64/boot/dts/qcom/sdm845.dtsi
index b1f6782a3ad9..d0151a204d4c 100644
--- a/arch/arm64/boot/dts/qcom/sdm845.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845.dtsi
@@ -977,6 +977,40 @@ q6routing: routing {
 						#sound-dai-cells = <0>;
 					};
 				};
+
+				q6mvm: apr-service@9 {
+					compatible = "qcom,q6mvm";
+					reg = <APR_SVC_ADSP_MVM>;
+
+					qcom,protection-domain = "avs/audio", "msm/adsp/audio_pd";
+
+					status = "disabled";
+
+					q6voicedai: dais {
+						compatible = "qcom,q6voice-dais";
+						#address-cells = <1>;
+						#size-cells = <0>;
+						#sound-dai-cells = <1>;
+					};
+				};
+
+				q6cvs: apr-service@a {
+					compatible = "qcom,q6cvs";
+					reg = <APR_SVC_ADSP_CVS>;
+
+					qcom,protection-domain = "avs/audio", "msm/adsp/audio_pd";
+
+					status = "disabled";
+				};
+
+				q6cvp: apr-service@b {
+					compatible = "qcom,q6cvp";
+					reg = <APR_SVC_ADSP_CVP>;
+
+					qcom,protection-domain = "avs/audio", "msm/adsp/audio_pd";
+
+					status = "disabled";
+				};
 			};
 
 			fastrpc {
-- 
2.55.0

